src/components/backend-ai-credential-list.ts

```diff
@@ -241,7 +241,22 @@ export default class BackendAICredentialList extends BackendAIPage {
             keypair['total_resource_slots'].cuda_shares = '-';
             keypair['total_resource_slots'].cuda_device = '-';
           }
-          ['cpu', 'mem', 'cuda_shares', 'cuda_device'].forEach((slot) => {
+          if ('rocm.device' in keypair['total_resource_slots']) {
+            keypair['total_resource_slots'].rocm_device = keypair['total_resource_slots']['rocm.device'];
+          }
+
+          if (('rocm_device' in keypair['total_resource_slots']) === false &&
+            keypair['default_for_unspecified'] === 'UNLIMITED') {
+            keypair['total_resource_slots'].rocm_device = '-';
+          }
+          if ('tpu.device' in keypair['total_resource_slots']) {
+            keypair['total_resource_slots'].rocm_device = keypair['total_resource_slots']['tpu.device'];
```

Comment:
`rocm_device` here should be changed to `tpu_device`?

Reply:
Fixed.